certified-connectors/1Me Corporate/apiDefinition.swagger.json

```diff
@@ -0,0 +1,179 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "1Me Corporate",
+    "description": "1Me is the easiest and fastest way to share your contact information.\nWith 1Me, you can have an unlimited number of contact cards. You can choose which contact information to include on each card, and then choose who to share each card with.\nThis is best for separating your business and personal contact information",
+
+    "contact": {
+      "name": "1Me Support",
+      "url": "https://www.1me.live/support",
+      "email": "support@1me.live"
+    },
+
+    "version": "1.0"
+  },
+  "host": "automate.1me.live",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/CardTempletes/": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@MICHAEL-MAURICE empty schema is not allowed. please define a proper schema. Please fix all schemas in the swagger file.